root/static/scripts/work.js

```diff
@@ -189,3 +189,19 @@ store.subscribe(renderWorkLanguages);
 renderWorkLanguages();
 
 MB.Control.initializeBubble('#iswcs-bubble', 'input[name=edit-work\\.iswcs\\.0]');
+
+let $typeBubble = $('#type-bubble')[0];
+let typeIdField = 'select[name=edit-work\\.type_id]';
+
+$('.type-bubble-description').hide();
```

Comment:
It seems to be more simple to use `style="display:none"` in the template instead.